components/camera/transformpipeline/mods.go

```diff
@@ -96,16 +95,16 @@ type resizeConfig struct {
 }
 
 type resizeSource struct {
-	originalStream gostream.VideoStream
-	stream         camera.ImageType
-	height         int
-	width          int
+	src    camera.VideoSource
```

Comment:
why not camera?

Reply:
I think the plan is for the next PR in this cleanup initiative to combine Camera and VideoSource where this will be addressed